components/link/link.js

```diff
@@ -3,6 +3,23 @@ import { css, html, LitElement } from 'lit';
 import { classMap } from 'lit/directives/class-map.js';
 import { FocusMixin } from '../../mixins/focus-mixin.js';
 import { ifDefined } from 'lit/directives/if-defined.js';
+import { isFocusVisibleSupported } from '../../helpers/focus.js';
+
+const linkFocusStyles = isFocusVisibleSupported() ? css`
+	.d2l-link:focus-visible {
+		border-radius: 3px;
+		outline: 2px solid var(--d2l-color-celestine);
+		outline-offset: 1px;
+		text-decoration: underline;
+	}
+` : css`
+	.d2l-link:focus {
+		border-radius: 3px;
+		outline: 2px solid var(--d2l-color-celestine);
+		outline-offset: 1px;
+		text-decoration: underline;
+	}
+`;
```

Comment:
Wonder if doing something like below might reduce the duplication and chance that we'll make a change to one but not the other one day? Maybe Lit won't even let you though.

```suggestion
const linkFocusPseudoClass = isFocusVisibleSupported() ? 'focus-visible' : 'focus';
const linkFocusStyles =  css`
	.d2l-link:${unsafeCSS(linkFocusPseudoClass)} {
		border-radius: 3px;
		outline: 2px solid var(--d2l-color-celestine);
		outline-offset: 1px;
		text-decoration: underline;
	}
`;
```

Reply:
Yep! That would be good, and I don't see any reason to avoid `unsafeCSS` in this case.

Reply:
I made this change with a separate commit since it required adding the `unsafeCSS` import as well.